hashintel/hash
# Add seed data for dev

## 🌟 What is the purpose of this PR?



This PR adds seeded dev data. The seeding happens automatically on first start if users do not already exist.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1201095311341924/1203106234191606/f) _(internal)_

## 🚫 Blocked by





## 🔍 What does this change?




- Adds org and page seeding

## 📜 Does this require a change to the docs?



- This happens automatically on dev environments, no need to document it.

## ⚠️ Known issues



- While Orgs and member users have been added in this PR, the Org functionality is currently borken. We're unable to fetch Org page trees and unable to display the org in the account dropdown. [Asana ticket](https://app.asana.com/0/1201095311341924/1203218567263548/f)
- The page ordering seems inconsistent even when we're setting the indices as expected. May not be set correct. [Asana ticket](https://app.asana.com/0/1201095311341924/1203225161554905/f)

## 🐾 Next steps



## 🛡 What tests cover this?




## ❓ How to test this?



1.  Checkout the branch, make sure to clear your external services `yarn external-services down` to clear DB (maybe also `docker volume prune`)
1.  Start `yarn dev`
1.  Log in to `alice` or `bob` accounts and see page trees filled from the start.

## 📹 Demo
